Community PR - Dune Dashboard (2a) (#5976)
w3f/polkadot-wiki: docs/general/dune-analytics/parachain-dashboards/nodle-dashboards.md

---
id: nodle-dashboards
title: Nodle Dashboards
sidebar_label: Nodle
description:

>   Nodle is a parachain on Polkadot focused on providing connectivity and data liquidity for the
>   Internet of Things (IoT). It enables devices to securely connect, interact, and transact with
>   efficiency.

Reviewer: ```suggestion
  Nodle is a parachain on Polkadot focused on providing connectivity and data liquidity for the
  Internet of Things (IoT).
```

keywords: [polkadot, dashboard, dune, nodle, IoT]
slug: ../nodle-dashboards
---

# Nodle Dashboards

## Overview

Nodle is a parachain on Polkadot focused on providing connectivity and data liquidity for the
Internet of Things (IoT). It enables devices to securely connect, interact, and transact with
efficiency.

## Featured Dashboards on Dune

> Here you'll find a variety of dashboards that help visualize data from the Nodle parachain:

Reviewer: ```suggestion
Here you will find a variety of dashboards that help visualize data from the Nodle parachain:
```

- [Nodle Dashboard](https://dune.com/substrate/nodle): A comprehensive view of IoT connectivity and
  transactions within the Nodle ecosystem.

Please also visit our dashboards for Nodle on
[Dune Analytics](https://dune.com/discover/content/relevant?q=title:Nodle%20author:substrate).

## Key Tables

Data from the Nodle parachain is organized into several key tables:

- `nodle.balances`
- `nodle.blocks`
- `nodle.calls`
- `nodle.events`,
- `nodle.extrinsics`
- `nodle.transfers`

## Useful Queries

> Currently, no specific queries have been provided. Please check back later for updates.

Reviewer: ```suggestion
Currently, no specific queries have been provided.
```
